internal/pkg/service/common/etcdop/op/atomic_test.go

```diff
@@ -69,10 +69,10 @@ func TestAtomicUpdate(t *testing.T) {
 			key3.Put(client, "value"),
 			op.Txn(client).
 				If(etcd.Compare(etcd.Value("key4"), "=", "value")).
-				Then(
+				Merge(
 					key5.Get(client),
 					op.Txn(client).
-						If(etcd.Compare(etcd.Value("checkMissing"), "=", "value")).
+						If(etcd.Compare(etcd.Version("missing"), "=", 0)).
 						Then().
```

Comment:
Improved test, to use `Merge` operation - because we use it more often, `if` condition now pass by default.